docs/CHANGELOG.md

```diff
@@ -30,6 +30,8 @@ The format is based on [Keep a Changelog][kc], and this project adheres to
 - ScreenDimensions now consistently reports window size in physical pixels. ([#1988])
 - `Config::load` now returns an error or failure rather than silently falling back to the default config. Same is true for the `from_config_file` methods on `RenderToWindow`, `WindowBundle`, and `WindowSystem` ([#1989])
 - Adds `get` methods to the underlying net::transport resources ([#2005])
+- Changed `SpriteSheetFormat::import_simple` to allow importing grid based `SpriteSheets` ([#2023])
+  Migration Note: Rons need to wrap their content in either Grid() or List()
```

Comment:
I like the change, but am wondering whether we should use `#[serde(untagged)]` instead. I don't have a strong preference, so am happy to merge in a few days if no one else does

Reply:
sry for my late reply. Do we have any drawbacks by using `#[serde(untagged)]`?

Reply:
It would leave ambiguity if there are future variants that have similar fields (and if they have the same fields, no way to tell serde which one to use). That said, it does boil down to how likely that's going to be. In my mind, `SpriteList` and `SpriteGrid` sufficiently cover all rectangular sprite use cases.

am happy with the tagged version (it helps discoverability), so shall merge.